app/src/main/java/com/android/sample/model/userAccount/UserAccountViewModel.kt

```diff
@@ -144,7 +146,7 @@ open class UserAccountViewModel(
       repository.sendFriendRequest(
           fromUser = currentUser,
           toUserId = toUserId,
-          onSuccess = {},
+          onSuccess = { getUserAccount(currentUser.userId) },
           onFailure = { exception ->
             Log.e("UserAccountViewModel", "Failed to send friend request", exception)
```

Comment:
Can you use a const into the scope of the class to define the TAG of the error "UserAccountViewModel" ?